ocs_ci/helpers/dr_helpers.py

```diff
@@ -718,3 +719,40 @@ def gracefully_reboot_ocp_nodes(namespace, drcluster_name):
     else:
         set_current_secondary_cluster_context(namespace)
     gracefully_reboot_nodes()
+
+
+def restore_backup():
+    """
+    Restores the backup in new hub and make it as active
+
+    """
+
+    restore_index = config.cur_index
+    config.switch_ctx(get_passive_acm_index())
+    backup_schedule = templating.load_yaml(constants.MDR_RESTORE_YAML)
+    backup_schedule_yaml = tempfile.NamedTemporaryFile(
+        mode="w+", prefix="restore", delete=False
+    )
+    templating.dump_data_to_temp_yaml(backup_schedule, backup_schedule_yaml.name)
+    run_cmd(f"oc create -f {backup_schedule_yaml.name}")
+    config.switch_ctx(restore_index)
```

Comment:
Need to make sure you are in the passive hub context here.

Reply:
done